refactor(protocol-designer): 8.4.4 release notes additions (#18225)
OpenTrons/opentrons-api: protocol-designer/release-notes.md

### Bug Fixes

- When you export a protocol with an unused staging area, gripper, or trash bin, Protocol Designer will include the fixture when your protocol is imported again.

> - When you zoom into a slot that has an older version of a labware definition, it no longer whitescreens.

Reviewer: ```suggestion
- Fixed a crash when zooming into a slot that has an older version of a labware definition.
```
"whitescreen" is kind of useful on the support/engineering side, but i think "crash" is more generally descriptive for users.

### Improved Features

> - Increased responsiveness and alignment with deck setup

Reviewer: alignment of what? some UX elements? let's say what they are.

Author: alignment of the timeline toolbox, step form toolbox, and the off-deck view. So does this work?

`Increased responsiveness and alignment with timeline toolbox, step form toolbox, and off-deck view of deck setup.`
